### DISCUSSION TITLE
Prisma + sqlite + blob → how?

### DISCUSSION THREAD

**Discussion by ruslankonev** [prisma/prisma#5891] [Q&A]
It is possible to write blobs with prisma into sqlite?

**Comment by ryands17** [comment]
@ruslankonev 👋 

Currently it's not possible to use Blobs with SQLite, so I would suggest creating a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look into this :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
